<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QDialog" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="bills">
      <attribute name="title">
       <string>Bills</string>
      </attribute>
      <widget class="QTableWidget" name="billsTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>10</y>
         <width>301</width>
         <height>431</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="sdssdfaf">
            <property name="text">
             <string>Bill ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="billEditLine"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QToolBox" name="toolBox">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="currentIndex">
           <number>5</number>
          </property>
          <widget class="QWidget" name="topup">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>217</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Top up</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>60</y>
              <width>110</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Amount</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="topupLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="topupButton">
               <property name="text">
                <string>Top up</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="withdraw">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>217</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Withdraw</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>60</y>
              <width>110</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Amount</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="withdrawLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="withdrawButton">
               <property name="text">
                <string>Withdraw</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="transfer">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>217</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Transfer</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>40</y>
              <width>110</width>
              <height>141</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>To (ID)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="transferToEditLine"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Amount</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="transferAmount"/>
             </item>
             <item>
              <widget class="QPushButton" name="transferButton">
               <property name="text">
                <string>Send</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="create">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>217</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Create </string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>90</x>
              <y>60</y>
              <width>110</width>
              <height>91</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Amount</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="createBillLineEdit"/>
             </item>
             <item>
              <widget class="QPushButton" name="createBillButton">
               <property name="text">
                <string>Create</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="block">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>217</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Block</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>90</y>
              <width>211</width>
              <height>26</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="blockButton">
               <property name="text">
                <string>Block</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="unblockButton">
               <property name="text">
                <string>Unblock</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
          <widget class="QWidget" name="frize">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>299</width>
             <height>217</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Freeze</string>
           </attribute>
           <widget class="QWidget" name="layoutWidget_3">
            <property name="geometry">
             <rect>
              <x>40</x>
              <y>80</y>
              <width>211</width>
              <height>26</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="pushButton_11">
               <property name="text">
                <string>Freeze</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_12">
               <property name="text">
                <string>Unfreeze</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="deposits">
      <attribute name="title">
       <string>Deposits</string>
      </attribute>
      <widget class="QToolBox" name="toolBox_2">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>10</y>
         <width>311</width>
         <height>461</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>311</width>
          <height>401</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Create</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>131</width>
           <height>291</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="depositAmount"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Stake</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="depositStake"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="deposit3">
            <property name="text">
             <string>3 M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="deposit6">
            <property name="text">
             <string>6 M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="deposit12">
            <property name="text">
             <string>12 M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="deposit24">
            <property name="text">
             <string>24 M</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="depositOther">
              <property name="text">
               <string>Other</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="depositOtherLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="CreateDepositButton">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>311</width>
          <height>401</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Withdraw</string>
        </attribute>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>120</y>
           <width>110</width>
           <height>152</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Deposit ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="withdrawDepositLineEdit"/>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Transfer to bill:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Bill ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="withdrawBillLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="withdrawDepositButton">
            <property name="text">
             <string>Withdraw</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QPushButton" name="depositsRefresh">
       <property name="geometry">
        <rect>
         <x>890</x>
         <y>570</y>
         <width>80</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="depositsTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="credits">
      <attribute name="title">
       <string>Credits</string>
      </attribute>
      <widget class="QTableWidget" name="creditsTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>651</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
      <widget class="QToolBox" name="toolBox_3">
       <property name="geometry">
        <rect>
         <x>670</x>
         <y>10</y>
         <width>311</width>
         <height>461</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>311</width>
          <height>401</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Create</string>
        </attribute>
        <widget class="QWidget" name="layoutWidget_2">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>131</width>
           <height>291</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="creditAmount"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Stake</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="creditStake"/>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="credit3">
            <property name="text">
             <string>3 M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="credit6">
            <property name="text">
             <string>6 M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="credit12">
            <property name="text">
             <string>12 M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="credit24">
            <property name="text">
             <string>24 M</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="creditOther">
              <property name="text">
               <string>Other</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="creditOtherLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="createCreditButton">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>311</width>
          <height>401</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Payment</string>
        </attribute>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>90</y>
           <width>110</width>
           <height>204</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Credit ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="creditId"/>
          </item>
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>From bill:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Bill ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="creditBill"/>
          </item>
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="creditAmountPay"/>
          </item>
          <item>
           <widget class="QPushButton" name="creditPaymentButton">
            <property name="text">
             <string>Pay</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
